deliravision/torch/models/basic_networks/segmentation.py

```diff
@@ -1,7 +1,7 @@
 import logging
 from abc import abstractmethod
 import torch
-from delira.models.abstract_network import AbstractPyTorchNetwork
+from delira.models.backends.torch.abstract_network import AbstractPyTorchNetwork
 
 
 class BaseSegmentationTorchNetwork(AbstractPyTorchNetwork):
```

Comment:
Do we still need this? Or will we provide another way to wrap pre-implemented segmentation networks?